### PR TITLE
Update Work History action formatting

### PR DESCRIPTION
**Pull request by shanear**
connects #1032

### CODE REVIEW

File: app/controllers/establish_claims_controller.rb [hunk 1]
```diff
@@ -50,6 +50,10 @@ def unprepared_tasks
 
   private
 
+  def tasks
```
Comment (shanear) [inline app/controllers/establish_claims_controller.rb]: Needed so we get a list of `EstablishClaim` objects and not `Task` objects. Which allows `EstablishClaim` to override `Task` methods like `joins_product`

File: app/models/tasks/establish_claim.rb [hunk 2]
```diff
@@ -201,6 +209,12 @@ def completion_status_after_review
 
     appeal.special_issues? ? :routed_to_ro : :routed_to_arc
   end
+
+  class << self
+    def joins_product
+      joins(:claim_establishment)
```
Comment (shanear) [inline app/models/tasks/establish_claim.rb]: prevents 10 extra SQL calls

File: app/models/tasks/establish_claim.rb [hunk 3]
```diff
@@ -91,6 +91,14 @@ def actions_taken
     ].reject(&:nil?)
   end
 
+  def completion_status_text
+    case completion_status
+    when "routed_to_ro" then "EP created for RO #{ep_ro_description}"
```
Comment (joofsh) [inline app/models/tasks/establish_claim.rb]: is this a ruby convention? This feels super hard to read imo. I would prefer them to be on separate lines, unless this is idiomatic

File: app/models/tasks/establish_claim.rb [hunk 4]
```diff
@@ -201,6 +209,12 @@ def completion_status_after_review
 
     appeal.special_issues? ? :routed_to_ro : :routed_to_arc
   end
+
+  class << self
+    def joins_product
```
Comment (joofsh) [inline app/models/tasks/establish_claim.rb]: love the join to save SQL calls, but why this method name? Why "product"? Is this short for end_product?

Comment (joofsh) [inline app/models/tasks/establish_claim.rb]: ahh, I see the comment in the parent class. Hmm

File: spec/models/task_spec.rb [hunk 5]
```diff
@@ -482,6 +483,26 @@ class FakeTask < Task
     end
   end
 
+  context ".completed_by" do
```
Comment (joofsh) [inline spec/models/task_spec.rb]: do we want a test for `joins_product`?

Comment (shanear) [inline spec/models/task_spec.rb]: It doesn't have any logic, it just saves some SQL, so it's hard to test optimizations. It's tested by integration tests too

File: app/models/tasks/establish_claim.rb (hunk 4 above)
Comment (shanear) [inline app/models/tasks/establish_claim.rb]: tried to pick a generic term for the output product of the task. What would you call it?

Here's some options I brainstormed:
`joins_output`
`joins_output_product`
`joins_result`
`joins_task_result`
`joins_task_data`

**Pull request merged**